wdl-grammar/src/v1/lint/preamble_comment.rs

```diff
@@ -135,7 +166,30 @@ version 1.0
         assert_eq!(warnings.len(), 1);
         assert_eq!(
             warnings.first().to_string(),
-            "[v1::W010::[Style]::Low] preamble comment without a double pound sign (1:1-1:12)"
+            "[v1::W010::[Style]::Low] preamble comment(s) without a double pound sign (1:1-1:12)"
+        );
+        Ok(())
+    }
+
+    #[test]
+    fn it_catches_badly_formatted_preamble_comment_block() -> Result<(), Box<dyn std::error::Error>>
+    {
+        let tree = Parser::parse(
+            Rule::document,
+            r#"# a comment
```

Comment:
So we're explicitly disallowing single pound comments in the preamble?

Reply:
Yes. That's not new in this PR. I thought we had a discussion about it back when this rule was first implemented.

Reply:
In the `document_preamble` lint check, I only look for `COMMENT` before the version. For some reason, I thought this rule (`preamble_comment`) was only about disallowing the double-pound syntax outside of the preamble.

Reply:
I don't think we should change it if we've discussed previously. I was just confirming that this was our intention here.

Reply:
The fact you had to ask to clarify whether this was an intended case or not is more evidence that we need to write a `body` (as defined in #12 ) for each rule. The current situation is that you need to code dive to discern what exactly a rule covers. We should have it written in plain text somewhere.